cmd/statusd/main.go

```diff
@@ -67,6 +90,30 @@ func main() {
 	node.Wait()
 }
 
+// startAPI starts the API server for remote control.
+func startAPI(backend *api.StatusBackend) error {
+	sepIdx := strings.LastIndex(*cliAddr, ":")
+	clientAddr := (*cliAddr)[:sepIdx]
+	port := (*cliAddr)[sepIdx+1:]
```

Comment:
I think [net.SplitHostPort()](https://golang.org/pkg/net/#SplitHostPort) does exactly the same.

Reply:
Iiiirks, looked for it but didn't found. Thanks. 🙄